concent_api/core/tests/test_core_views.py

```diff
@@ -355,6 +355,56 @@ def test_receive_return_http_204_if_no_messages_in_database(self):
         self.assertEqual(response.content.decode(), '')
         assert len(ReceiveStatus.objects.filter(delivered=False)) == 0
 
+    @freeze_time("2017-11-17 12:00:00")
```

Comment:
Please move this inside the function. You only need to freeze time around the `client.post()` call and I think this makes the test easier to undrstand because you can read the timestamps in order.